common/app/model/CardStylePicker.scala

```diff
@@ -11,22 +11,33 @@ object CardStylePicker {
 
   def apply(content: CapiContent): CardStyle = {
     val tags = content.tags.map(_.id).toSeq
-    extractCampaigns(tags) match {
-      case Nil                                                                         => CardStyle(content, TrailMetaData.empty)
-      case campaign :: Nil if campaign.id.toString.toLowerCase() == "specialreportalt" => SpecialReportAlt
-      case _                                                                           => SpecialReport
+    val campaigns = extractCampaigns(tags)
+    campaigns match {
+      case Nil => CardStyle(content, TrailMetaData.empty)
+      case _   => getCardStyleForReport(campaigns)
     }
   }
 
   def apply(content: FaciaContent): CardStyle = {
     val tags = FaciaContentUtils.tags(content).map(_.id)
-    extractCampaigns(tags) match {
-      case Nil                                                                         => FaciaContentUtils.cardStyle(content)
-      case campaign :: Nil if campaign.id.toString.toLowerCase() == "specialreportalt" => SpecialReportAlt
-      case _                                                                           => SpecialReport
+    val campaigns = extractCampaigns(tags)
+    campaigns match {
+      case Nil => FaciaContentUtils.cardStyle(content)
+      case _   => getCardStyleForReport(campaigns)
     }
   }
 
+  def getCardStyleForReport(campaigns: List[Campaign]): CardStyle = {
+    if (containsSpecialReportAltCampaign(campaigns)) SpecialReportAlt else SpecialReport
+  }
+
+  private def containsSpecialReportAltCampaign(campaigns: List[Campaign]): Boolean = {
+    val specialReportAltCampaigns = campaigns.filter(campaign =>
+      campaign.fields.asInstanceOf[ReportFields].campaignId.toLowerCase() == "specialreportalt",
```

Comment:
We are only interested in ReportFields because SpecialReportAlt and SpecialReport are campaigns of type Report. See[ code for Fields in the targeting-client](https://github.com/guardian/targeting-client/blob/main/src/main/scala/com/gu/targeting/client/Fields.scala#L11).